README.md

```diff
@@ -59,75 +74,126 @@ The code is in maintenance mode with no significant active development planned.
-title="More information in the wiki"><img src="https://ds0k0en9abmn1.cloudfront.net/static/charts/images/tx-logo-micro.646b0065fce6.png" ></a></td>
+    <td>
+      Translations
+    </td>
+    <td colspan=2>
+      <a target="_blank" style="text-decoration:none; color:black; font-size:66%" href="https://github.com/gitextensions/gitextensions/wiki/Translations" title="More information in the wiki"><img src="https://ds0k0en9abmn1.cloudfront.net/static/charts/images/tx-logo-micro.646b0065fce6.png" ></a>
+    </td>
   </tr>
 </table>
 
 
 ## Downloads
-<img src="https://img.shields.io/github/downloads/gitextensions/gitextensions/total.svg?maxAge=86400" alt="GitHub Releases (latest)"></a>
+<a href="https://github.com/gitextensions/gitextensions/releases" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/github/downloads/gitextensions/gitextensions/total.svg?label=GitHub%20downloads%20(total)&cacheSeconds=86400"></a> <a href="https://sourceforge.net/projects/gitextensions/files" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/sourceforge/dt/gitextensions.svg?label=SourceForge%20downloads%20(total)&cacheSeconds=86400"></a> <a href="https://chocolatey.org/packages/gitextensions" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/chocolatey/dt/gitextensions.svg?label=Chocolatey%20downloads%20(total)&cacheSeconds=86400"></a>
 
 <table>
   <tr>
-    <td><strong>v3.1</strong></td>
-    <td colspan=2><a href="https://github.com/gitextensions/gitextensions/releases/latest#hovercard-aria-description">[ Download ]</a><br />
-        GitHub: <a href="https://github.com/gitextensions/gitextensions/releases/latest#hovercard-aria-description" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/github/downloads/gitextensions/gitextensions/latest/total.svg?maxAge=86400" alt="GitHub Releases (latest)"></a> | SourceForge: <a href="https://sourceforge.net/projects/gitextensions/" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/sourceforge/dm/gitextensions.svg" alt="SourceForge"></a></td>
+    <td>
+      <strong>v3.1</strong>
+    </td>
+    <td>
+      <a href="https://github.com/gitextensions/gitextensions/releases/latest">[ Download ]</a><br />
+      <a href="https://github.com/gitextensions/gitextensions/releases/latest" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/github/downloads/gitextensions/gitextensions/latest/total.svg?label=GitHub%20downloads%20(latest)&cacheSeconds=3600"></a> <a href="https://sourceforge.net/projects/gitextensions/" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/sourceforge/dt/gitextensions/v3.1.svg?label=SourceForge%20downloads%20(latest)&cacheSeconds=3600"></a>
```

Comment:
Unfortunately, the badge `SourceForge doanloads (latest)` is not a true _latest_. It carries a version number and needs to be bumped with every release: `https://img.shields.io/sourceforge/dt/gitextensions/v3.1.svg`.

Reply:
oh well, there are few things like that.
we'll need to document them in the wiki - https://github.com/gitextensions/gitextensions/wiki/How-To%3A-prepare-a-release

Reply:
done